src/routes/user.routes.js

```diff
@@ -11,8 +11,8 @@ router.use(function timeLog(req, res, next) {
 
 router
   .get("/:id", userStorage.getUser)
-  .put("/:id", userStorage.updateUser)
-  .delete("/:id", userStorage.deleteUser);
+/*  .put("/:id", userStorage.updateUser)
```

Comment:
You can delete the comment from this file if we don't support anymore those operations.